docs/releases/v2019.1.rst

```diff
@@ -0,0 +1,219 @@
+If enabled, by setting ``wifi5.outdoor_chanlist``, a number of devices that are commonly installed outdoors will have outdoor mode automatically enabled during their initial setup, specifically:
+
+* Ubiquiti
+
+    - Bullet M
+    - Litebeam M5
+    - Nanostation M5
+    - Nanostation M5 Loco
+    - Rocket M5
+    - Rocket M5 TI
+    - Unifi AC Mesh
+    - Unifi AC Mesh Pro
+    - Unifi Outdoor
+
+* TP-Link
+
+    - CPE510
+    - WBS510
+
+See the :ref:`wifi5 <user-site-wifi5>` section for the *site.conf* configuration of this feature.                  
+
+
+Hoodselector: Geolocation Mode
+==============================
+
+Hoodselector is a new concept for continuous reevaluation of the node's domain. As a first step we support its geolocation feature. It can be used to automatically switch nodes during their runtime between domains.
+
+See the :doc:`../package/gluon-hoodselector` documentation for details.
+
+
+Device Deprecation
+==================
+
+The ar71xx-tiny and several devices in the ramips-rt305x target have been marked as deprecated. The `GLUON_DEPRECATED` flag was introduced to offer downstream communities the choice on how to deal with the ending support for those devices. Devices or targets marked as deprecated will very likely not be included in following Gluon releases anymore, usually due to their insufficient flash size.
+
+See the :ref:`Build configuration <user-site-build-configuration>` section for details.
+
+
+x86 images support firstboot
+============================
+
+x86 images are now using squashfs over ext4 and can now have their configuration reset by using ``firstboot``.
+
+
+Bugfixes
+********
+
+* Fixes passwordless SSH access when gluon-authorized-keys was used without gluon-setup-mode. (`#1777 <https://github.com/freifunk-gluon/gluon/issues/1777>`_)
+
+- Fixes cross-domain leakage of respondd data by not joining the link-local multicast group on br-client. Nodes now only listen on ``[ff05::2]:1001`` for respondd queries. This may require an update to your respondd querying tooling. (`#1701 <https://github.com/freifunk-gluon/gluon/issues/1701>`_)
+
+
+Site changes
+************
+
+When updating a site configuration from Gluon 2018.2.x, the following changes
+must be made:
+
+site.mk
+=======
+
+- We now require the ``GLUON_DEPRECATED`` variable to be set to decide how to handle the image generation for deprecated devices. (`#1745 <https://github.com/freifunk-gluon/gluon/pull/1745>`_)
+
+- The variable ``DEVICES``, that controls which devices to build images for, has been renamed to ``GLUON_DEVICES``. (`#1686 <https://github.com/freifunk-gluon/gluon/pull/1686>`_)
+
+site.conf
+=========
+
+- The options ``wifi*.basic_rates`` and ``wifi*.supported_rates`` have been removed, as the legacy 802.11b rates are now disabled by default. (`#1716 <https://github.com/freifunk-gluon/gluon/pull/1716>`_)
+
```

Comment:
Isn't the addition of
```
mesh = {
  batman_adv = {
    routing_algo = 'BATMAN_IV',
  }
}
```
also needed?

IIRC this was needed when I tried to compile gluon this week.

Reply:
Just checked (to be really sure). With unmodified site.conf I get this output only with "make -j1 V=s"
```
cp -fpR /home/txtfile/gluon_dev/openwrt/build_dir/target-i386_pentium4_musl/root-x86 /home/txtfile/gluon_dev/openwrt/build_dir/target-i386_pentium4_musl/root.orig-x86
*** site.conf / domains/eventnet.conf error: expected mesh.batman_adv.routing_algo to be one of the given array ["BATMAN_IV", "BATMAN_V"], but it is unset
postinst script ./usr/lib/opkg/info/gluon-mesh-batman-adv-15.postinst has failed with exit code 1
make[3]: *** [package/Makefile:71: package/install] Error 1
make[3]: Leaving directory '/home/txtfile/gluon_dev/openwrt'
make[2]: *** [package/Makefile:108: /home/txtfile/gluon_dev/openwrt/staging_dir/target-i386_pentium4_musl/stamp/.package_install] Error 2
make[2]: Leaving directory '/home/txtfile/gluon_dev/openwrt'
make[1]: *** [/home/txtfile/gluon_dev/openwrt/include/toplevel.mk:218: world] Error 2
make[1]: Leaving directory '/home/txtfile/gluon_dev/openwrt'
make: *** [Makefile:151: all] Error 2
```

Reply:
Good catch, added.